src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -86,49 +87,56 @@ public void LogFunctionExecutionEvent(string executionId, string siteName, int c
                     WriteEvent(57909, executionId, siteName, concurrency, functionName, invocationId, executionStage, executionTimeSpan, success);
                 }
             }
+        }
+
+        [EventSource(Guid = "a7044dd6-c8ef-4980-858c-942d972b6250")]
+        [SuppressMessage("Microsoft.StyleCop.CSharp.NamingRules", "SA1306:FieldNamesMustBeginWithLowerCaseLetter", Justification = "Need to use Pascal Case for MDS column names")]
+        public class FunctionsSystemLogsEventSource : EventSource
```

Comment:
Events using FunctionsEventSource are used in Antares for billing, metrics and pulse. Unfortunately these events cannot be moved to new GUID as the EventIds and GUID are hardcoded for event trace watcher in Antares.  If any of these events needs to be updated, MDS config needs to be updated as well.